### ISSUE TITLE
ImportError: cannot import name 'PubSubManager'

### ISSUE THREAD

**Issue by benjaminturley** [miguelgrinberg/Flask-SocketIO#465]
Getting this when I try to run my basic server with eventlet. Tried uninstall which did not work. The app was working fine an hour ago until windows restarted for an update after which it all came crashing down. I changed nothing in the init or run.py files.

```
(flask) C:\Users\usr\Documents\BitBucket\site\site>python run.py
Traceback (most recent call last):
  File "run.py", line 2, in <module>
    from app import app, socketio
  File "C:\Users\usr\Documents\BitBucket\site\site\app\__init__.py", line 4, in <module>
    from flask_socketio import SocketIO
  File "C:\Users\usr\Documents\BitBucket\site\flask\lib\site-packages\flask_socketio\__init__.py", line 16, in <module>
    import socketio
  File "C:\Users\usr\Documents\BitBucket\site\flask\lib\site-packages\socketio\__init__.py", line 5, in <module>
    from .pubsub_manager import PubSubManager
ImportError: cannot import name 'PubSubManager'
```

**Comment by miguelgrinberg** [comment]
Reinstall the python-socketio package. It's probably broken. Or better yet, recreate your entire virtualenv.

**Comment by benjaminturley** [comment]
Okay, the problem went away but the solution was a bit unfortunate. Tried reinstalling the socketio package which didn't work. I then recreated the virtualenv which didn't work either. I ended up having to pull a backup from version control and set up in a different directory before moving the files back. If anyone stumbles upon this problem in the future, you might need to nuke the storage location to fix it.
